ui/appui-react/src/appui-react/framework/FrameworkChildWindows.ts

```diff
@@ -11,6 +11,7 @@ export interface OpenChildWindowInfo {
   childWindowId: string;
   window: Window;
   parentWindow: Window;
+  root?: any;
```

Comment:
Can you move this into a different structure that will remain private, I'd rather not exposer this until we have someone that actually have a need for it. (I know I said it should be in childWindowInfo, but I didnt realise this was part of our public API, only that it was a structure "per" window.

Reply:
Should be good now!

Reply:
We are no longer using it, but it is still in the interface :)

Reply:
My bad, it's gone now